dbc2val/test/test_mapping_transform/test.json

```diff
@@ -0,0 +1,242 @@
+{
+  "A": {
+    "children": {
+      "MappingIntInt": {
+        "datatype": "string",
+        "dbc": {
+          "$file_name$": "test.vspec",
+          "$line$": 114,
+          "on_change": true,
+          "signal": "S5",
+          "transform": {
+            "$file_name$": "test.vspec",
+            "$line$": 117,
+            "mapping": [
+              {
+                "$file_name$": "test.vspec",
```

Comment:
Are the $file_name$ things an artifact from the vss-tools? It looks like those "scratchpad" data the parser attaches for giving better error messages. Do we consider it a bug upstream and will fix it there? Because in "vanilla" VSS vspec these things are cleaned up.